packages/grpc-native-core/src/common.js

```diff
@@ -75,6 +77,73 @@ exports.defaultGrpcOptions = {
   deprecatedArgumentOrder: false
 };
 
+/**
+ * Create an Error object from a status object
+ * @param {grpc~StatusObject} status The status object
+ * @return {Error} The resulting Error
+ */
+exports.createStatusError = function(status) {
+  let statusName = _.invert(constants.status)[status.code];
+  let message = `${status.code} ${statusName}: ${status.details}`;
+  let error = new Error(message);
+  error.code = status.code;
+  error.metadata = status.metadata;
+  error.details = status.details;
+  return error;
+};
+
+/**
+ * Get a method's type from its definition
+ * @param {grpc~MethodDefinition} method_definition
+ * @return {number}
+ */
+exports.getMethodType = function(method_definition) {
+  if (method_definition.requestStream) {
+    if (method_definition.responseStream) {
+      return constants.methodTypes.BIDI_STREAMING;
+    } else {
+      return constants.methodTypes.CLIENT_STREAMING;
+    }
+  } else {
+    if (method_definition.responseStream) {
+      return constants.methodTypes.SERVER_STREAMING;
+    } else {
+      return constants.methodTypes.UNARY;
+    }
+  }
+};
+
+/**
+ * Get a call object built with the provided options.
+ * @param {grpc.Channel} channel
+ * @param {string} path
+ * @param {grpc.Client~CallOptions=} options Options object.
+ */
+exports.getCall = function(channel, path, options) {
```

Comment:
It looks like this function is only used in `client_interceptors.js`, so it should be defined there.